example/index.tsx

```diff
@@ -189,6 +203,22 @@ function MainForm({ subscription }: any) {
 	};
 
 	const formFields = [
+		<Autocomplete
+			label="Pick at least one planet"
+			name="planet"
+			required={required.planet}
+			options={autocompleteData}
+			getOptionValue={option => option.value}
+			getOptionLabel={option => option.label}
+			disableCloseOnSelect={true}
+			renderOption={(option, { selected }) => (
+				<React.Fragment>
```

Comment:
stupid minor nit, but i'm ok with `<>`

Reply:
haha I normally use `<>` as well but copied it from that example 😏